content/themes/semantic-ui/twitter/index.slim

```diff
@@ -124,4 +124,3 @@
       == @items['/partials/examples/transition.slim'].compiled_content
 
 link href="/static/kitchen-sink/stylesheets/kitchen-sink.css" rel="stylesheet" type="text/css"
-script src="/static/kitchen-sink/javascripts/kitchen-sink.js"
```

Comment:
OK, I will try to move the progress part of JS code somewhere else and put kitchen-sink.js back.